src/app/pages/home/home-routing.module.ts

```diff
@@ -11,44 +11,25 @@ const routes: Routes = [
     component: HomePage,
     children: [
       {
-        path:'rolodex',
-        children: [
-          {
-            path:'',
-            loadChildren:'./rolodex/rolodex.module#RolodexPageModule'
-          }
-        ]
+        path:'rolodex', loadChildren:'./rolodex/rolodex.module#RolodexPageModule'
       },
 
       {
-        path:'exchange',
-        children: [
-          {
-            path:'',
-            loadChildren:'./exchange/exchange.module#ExchangePageModule'
-          }
-        ]
+        path:'exchange', loadChildren:'./exchange/exchange.module#ExchangePageModule'
       },
 
       {
-        path:'profile',
-        children: [
-          {
-            path:'',
-            loadChildren:'./profile/profile.module#ProfilePageModule'
-          }
-        ]
+        path:'profile', loadChildren:'./profile/profile.module#ProfilePageModule'
       },
       {
-        path: '',
-        redirectTo: 'rolodex',
+        path: '', redirectTo: 'rolodex',
         pathMatch: 'full'
       }
     ]
 },
   {
     path: '',
-    redirectTo: 'rolodex',
+    redirectTo: 'home/rolodex',
```

Comment:
I'm trying not to refer to the main routing paths since we have a local path here. I am considering removing the references to these nested routes in the main routing module (since any other page can simply navigate to `home`) but more research is needed.

Reply:
Other pages can, but I think the difference is having the tabs "dashboard" present globally on each page.